testdata/protobuf/dependencies/main.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+import "dep.proto";
```

Comment:
I know this seems like it might not matter, but can you do a case where the imports are in another directory?

Reply:
Would a subdirectory suffice for the case you're trying to catch or do you want to test something with multiple import paths? Eg `main.proto`, `dependencies/dep.proto` with
-  `import "dependencies/dep.proto"` in main or
- `import "dep.proto"` and protoc `-I dependencies/`

Reply:
Both would be great, but the latter is my primary concern I think.

Reply:
Added test cases for both.